src/components/molecules/Calender/ModalRegistSchedule.vue

```diff
@@ -43,13 +43,26 @@ import InputTime from "@/components/atoms/InputTime.vue";
 export default {
   name: "CalenderModalRegistSchedule.vue",
   components: { InputDate, InputTime },
+  props: {
+    isScheduleRegistModalOpen: {

```

Comment:
```suggestion
    isModalOpen: {
```
このコンポーネントはそもそもscheduleRegisterModalであることはわかってるので、この名前だと冗長だと感じました。
シンプルでも良いかと。

これが悪いわけでもないですが、こっちでも良いのではってぐらいです